truss/templates/triton/model/triton_model_wrapper.py

```diff
@@ -0,0 +1,80 @@
+import importlib
+import inspect
+import sys
+from pathlib import Path
+from typing import Dict
+
+import triton_python_backend_utils as pb_utils
+from shared.secrets_resolver import SecretsResolver
+from transform import transform_pydantic_to_triton, transform_triton_to_pydantic
+from utils.pydantic import inspect_pydantic_model
+from utils.signature import signature_accepts_keyword_arg
+
+
+class ModelWrapper:
+    def __init__(self, config: Dict):
+        self._config = config
+        self._model = None
+        self._input_type = None
+        self._output_type = None
+        self._input_type_fields = None
+        self._output_type_fields = None
+
+    def instantiate(self):
```

Comment:
`instantiate` is a confusing method name here. It sounds like a static method that instantiates a new instance of the class it's defined in. Maybe `initialize`?